src/transformers/pipelines/token_classification.py

```diff
@@ -179,6 +189,30 @@ def _sanitize_parameters(
             postprocess_params["aggregation_strategy"] = aggregation_strategy
         if ignore_labels is not None:
             postprocess_params["ignore_labels"] = ignore_labels
+        if process_all is True:
+            if self.tokenizer.is_fast:
+                if stride is not None:
+                    if stride < 0:
+                        stride = 0
```

Comment:
Don't sanitize here.

negative stride is a bug, but it's not your error to handle, this is a tokenizer level bug.